docs/development-api/github.md

```diff
@@ -128,7 +128,7 @@ and maps them to GitHub GraphQL [FileChanges].
 ```typescript
 import { GitHub } from 'skuba';
 
-const fileChanges = await GitHub.readFileChanges([
+const fileChanges = await GitHub.readFileChanges(dir, [
```

Comment:
`ChangedFiles` are relative to `dir` so we really need both to correctly evaluate `FileChanges`